datasets/huggingface-mnist/metadata.json

```diff
@@ -0,0 +1,75 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "applyTransform": "ml:applyTransform",
+    "csvColumn": "ml:csvColumn",
+    "data": {
+      "@id": "ml:data",
+      "@type": "@json"
+    },
+    "dataExtraction": "ml:dataExtraction",
+    "dataType": {
+      "@id": "ml:dataType",
+      "@type": "@vocab"
+    },
+    "field": "ml:field",
+    "fileProperty": "ml:fileProperty",
+    "format": "ml:format",
+    "includes": "ml:includes",
+    "jsonPath": "ml:jsonPath",
+    "ml": "http://mlcommons.org/schema/",
+    "path": "ml:path",
+    "recordSet": "ml:recordSet",
+    "references": "ml:references",
+    "regex": "ml:regex",
+    "replace": "ml:replace",
+    "sc": "https://schema.org/",
+    "separator": "ml:separator",
+    "source": "ml:source",
+    "subField": "ml:subField",
+    "wd": "https://www.wikidata.org/wiki/"
+  },
+  "@type": "sc:Dataset",
+  "name": "simple-parquet",
+  "description": "Example to read Parquet files.",
```

Comment:
Related to https://github.com/mlcommons/croissant/discussions/58, that's maybe something we would want in another field.
What about having here the huggingFace description of the dataset?